src/components/Person.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Person as PersonType } from '../types/Person';
+import { PersonLink } from './PersonLink';
+import { useParams } from 'react-router-dom';
+
+interface Props {
+  person: PersonType;
+}
+
+export const Person: React.FC<Props> = ({ person }) => {
+  const { sex, born, died, fatherName, motherName, mother, father } = person;
```

Comment:
Ensure that the `Person` type includes `fatherName` and `motherName` properties. If these properties are not part of the `Person` type, consider updating the type definition or adjusting the code to align with the available properties.